src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -1321,6 +1322,86 @@ internal static object ContainsOperator(ExecutionContext context, IScriptExtent
             return BoolToObject(!contains);
         }
 
+        internal static bool AnyOperator(
+            ExecutionContext context,
+            CallSite<Func<CallSite, object, IEnumerator>> getEnumeratorSite,
+            CallSite<Func<CallSite, object, object, object>> comparerSite,
+            object left,
+            object right)
+        {
+            ScriptBlock predicate = right as ScriptBlock;
+            if (predicate == null)
+            {
+                return ContainsOperatorCompiled(context, getEnumeratorSite, comparerSite, left, right);
+            }
+
+            IEnumerator list = getEnumeratorSite.Target.Invoke(getEnumeratorSite, left);
+            if (list == null || list is EnumerableOps.NonEnumerableObjectEnumerator)
+            {
+                object result = predicate.DoInvokeReturnAsIs(true, ScriptBlock.ErrorHandlingBehavior.WriteToCurrentErrorPipe, left, null, null, null);
```

Comment:
```suggestion
                object result = predicate.DoInvokeReturnAsIs(
                    true,
                    ScriptBlock.ErrorHandlingBehavior.WriteToCurrentErrorPipe,
                    left,
                    null,
                    null,
                    null);
```

Also, please name the true / null arguments here for clarity.